README.md

```diff
@@ -1,7 +1,11 @@
 # NameLayer
 
-See the wiki for details on group creation, management, and other commands.
+A Citadel reinforcement group manager. Allows the creation of groups in Minecraft. From buildings to chat messages, one can selectivity restrict or expand which players are on group and their level of access.
```

Comment:
NameLayer has nothing to do with Citadel, its just the group manager. Citadel is a plugin using this group functionality, just like Bastion, CivChat2 or JukeAlert and shouldn't be used  as primary way to describe NameLayer

Reply:
Whoops, I was mirroring the repo description which is "A Citadel reinforcement group manager. Allows the creation of groups in Minecraft. Built for Spigot 1.10 http://www.devotedmc.com".

The wiki link still needs to be updated as the current is broken.